### PR TITLE
Add documentation for user profiles

### PR DESCRIPTION
**Pull request by rdswift**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

New "user profiles" feature added to Picard 2.7.

### Description of the Change

Documentation for the new feature.

### Additional Action Required

Once approved, will need translation files updated.

### CODE REVIEW

**Review by phw — COMMENTED** [review]
Looks good to me, but I'd wait with merging this until https://github.com/metabrainz/picard/pull/1851 has been merged as well.

File: usage/user_profiles.rst [hunk 1]
```diff
@@ -105,5 +105,5 @@ To update the settings for the options managed by the new profile, the recommend
 
 1. Enable the new profile and disable all others.  This is the safest way to avoid accidentally applying changes to another profile.
 2. Exit the profile editor and open the :menuselection:`Options...` screen.
-3. Make the desired changes to the options managed by the profile, and save them using the :kbd:`Make_It_So!` button.
+3. Make the desired changes to the options managed by the profile, and save them using the :kbd:`Make It So!` button.
```
Comment (phw) [inline usage/user_profiles.rst:108]: I must say  I liked the old display better. I prefer

grafik

over 

grafik

I think what triggered the change is the use here, but `:kbd:` shouldn't be used here any way. `:kbd:` is specifically about keyboard short cuts. In this case [`:guilabel:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel) would be the appropriate role I think.

**Review by phw — CHANGES_REQUESTED** [review]

File: usage/user_profiles.rst (hunk 1 above)
Comment (zas) [reply usage/user_profiles.rst:108]: I prefer the first one too

Comment (rdswift) [reply usage/user_profiles.rst:108]: As do I.

**Review by phw — APPROVED** [review]

**Pull request merged**